engine/client/client.go

```diff
@@ -323,6 +323,9 @@ func (c *Client) startEngine(ctx context.Context) (rerr error) {
 	if c.CloudCallback != nil {
 		if url, msg, ok := enginetel.URLForTrace(ctx); ok {
 			c.CloudCallback(ctx, url, msg)
+		} else {
+			// fallback to quickstart flow if the user is not logged in.
+			c.CloudCallback(ctx, "https://dagger.cloud/traces/"+uuid.NewString(), "logged out")
```

Comment:
This is really weird from a usability perspective - my initial read of this PR was that we were uploading *all* traces to dagger cloud, and then allowing users to create an organization with them in here. For this case, we shouldn't be printing a URL that looks *anything* like a traces view URL, it definitely confused me - it's not a real trace URL, so we shouldn't pretend it is.

Reply:
Also note, it's slightly weird that we show the "logged out" message, at the end of a trace in `tui` mode if you're not logged in, but that you don't get the full trace URL if you are logged in?

We should balance that.

Reply:
> it's not a real trace URL, so we shouldn't pretend it is.

 cc @shykes

Reply:
As discussed in the issue - the logged in experience did show the Trace URL when we started working on this. But it has since been removed.

Reply:
resolving this as I've removed this code since it requires further discussion